pkg/controller/gc.go

```diff
@@ -525,3 +532,16 @@ func (c *Controller) gcStaticRoute() error {
 	}
 	return nil
 }
+
+func (c *Controller) isOVNProvided(providerName string, pod *corev1.Pod) (bool, error) {
+	ls := pod.Annotations[fmt.Sprintf(util.LogicalSwitchAnnotationTemplate, providerName)]
+	subnet, err := c.config.KubeOvnClient.KubeovnV1().Subnets().Get(context.Background(), ls, metav1.GetOptions{})
+	if err != nil {
+		klog.Errorf("parse annotation logical switch %s error", ls)
```

Comment:
We should log error here to indicate what happended.

Reply:
right. error info added